.teamcity/settings.kts

```diff
@@ -52,11 +53,6 @@ object ChocolateyGUI : BuildType({
             }
         }
 
-        step {
-            name = "Include Signing Keys"
-            type = "PrepareSigningEnvironment"
-        }
-
         script {
             name = "Call Cake"
             scriptContent = """
```

Comment:
In the line below this comment, can you change `build.official.bat` to `build.bat`?  Since the "Include Signing Keys" step has been removed, we shouldn't call `build.official.bat` anymore.

Reply:
@corbob @gep13 - is the signing prep step not still required here? Without it the GUI build will never be able to sign things (with any cert)... especially tagged builds.